addons/ConsoleBG/ConsoleBG.lua

```diff
@@ -28,19 +29,107 @@ config.register(settings, function(settings)
     windower.prim.set_size('ConsoleBG', settings.extents.x, settings.extents.y)
 end)
 
+windower.register_event('addon command', function(command1, command2, command3)
```

Comment:
It's generally more useful to only specify the root command and leave the others variable (as `...`), then pack them into an `args` table or something. That way it's easier to amend in the future.